object_store_factory/src/aws.rs

```diff
@@ -1,3 +1,4 @@
+use object_store::aws::S3ConditionalPut;
 use object_store::{
     aws::resolve_bucket_region, aws::AmazonS3Builder, aws::AmazonS3ConfigKey, path::Path,
     ClientConfigKey, ClientOptions, ObjectStore,
```

Comment:
Just to keep this more organized:

```suggestion
use object_store::aws::{AmazonS3Builder, AmazonS3ConfigKey, S3ConditionalPut, resolve_bucket_region};
use object_store::{ ClientConfigKey, ClientOptions, ObjectStore };
use object_store::path::Path;
```

Reply:
Thanks, good catch.